meadServer/app.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
```
console.log('Error');
console.error(err);
```
Seems like these should be the same severity level.  You can probably just do the second level